src/components/CookieWarning.js

```diff
@@ -36,21 +33,28 @@ export const CookieWarning = () => {
     }, cookies)
     signOut()
   }
-  return !(acceptedCookies) && div({
+  return !cookiesAccepted && div({
     style: {
       position: 'fixed', height: 100, bottom: 0, width: '100%',
       display: 'flex', alignItems: 'center', justifyContent: 'space-between',
-      backgroundColor: 'white', borderTop: `6px solid ${colors.primary()}`
+      backgroundColor: colors.dark(0.15), borderTop: `6px solid ${colors.primary()}`
```

Comment:
(especially) because I know you care about A11Y, I think the contrast suffers a bit with the darker background color -- lighthouse flags this as an issue with the Reject button and the privacy policy link.

![Screen Shot 2021-01-29 at 6 09 36 PM](https://user-images.githubusercontent.com/5375000/106336619-47bd4900-625d-11eb-93b3-e676210d2bfb.png)